lib/linguist/languages.yml

```diff
@@ -2083,20 +2083,46 @@ JSON:
   filenames:
   - ".arcconfig"
   - ".htmlhintrc"
-  - ".jscsrc"
-  - ".jshintrc"
   - ".tern-config"
   - ".tern-project"
   - composer.lock
   - mcmod.info
   language_id: 174
-JSON5:
+JSON with comments:
```

Comment:
Should we consider this a title and use title case (so, `JSON with Comments`)? It feels weird every time I open this pull request...

Reply:
I agree with @pchaigno: this needs to be title case.